src/package.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Package xmlns="http://soap.sforce.com/2006/04/metadata">
+	<types>
```

Comment:
Package should contain all the metadata you would like to deploy. For instance lightning components. You could use the wildcard "*" in members tag to deploy the files instead of specify each of them.